app/views/users/show.html.erb

```diff
@@ -1,6 +1,42 @@
 <section class="user-section">
   <h2><%= "Name: #{@user.name}" %></h2>
   <article class="timeline">
+  <div class="lists">
+  <article class="timeline">
+      <h3>Friends:</h3>
+      <% @friends.each do |friend| %>
+        <ul class="posts">
+          <li><%= link_to friend.name, friend, class: 'name-links' %>  
+            <%#= link_to 'Unfriend', user_unfriend_path({friend_id: friend.id, user_id: current_user.id}) %>
+          </li>
+        </ul>
+      <% end %>  
+    </article>
+  <% if actual_user %>
+    <article class="timeline">
+      <h3>Pending Friends:</h3>
+      <% @pending_friends.each do |friend| %>
+        <ul class="posts">
+          <li><%= link_to friend.name, friend, class: 'name-links' %></li>
+        </ul>
+      <% end %>  
+    </article>
+
+    <article class="timeline">
+      <h3>Friends Requests:</h3>
+      <% @friend_requests.each do |friend| %>
+        <ul class="posts">
+          <li> <%= link_to friend.name, friend, class: 'name-links' %> : 
+           <%= link_to 'Accept Friend Request', user_create_friend_path({friend_id: friend.id, user_id: current_user.id}), class: 'link-btn' %> &nbsp; 
+           <%= link_to 'Reject Friend Request', user_reject_friend_request_path({friend_id: friend.id, user_id: current_user.id}), class: 'link-btn2' %> 
+          </li>
+        </ul>
+        <br>
+      <% end %>  
+    </article>
+
+  <% end %>
```

Comment:
- [ ] Place the if statement in a helper method that you call here and within the helper method render a partial where you put all the markup that is contained in the if statement since there should be no logic in views